api/package-lock.json

```diff
@@ -1,8 +1,4401 @@
 {
   "name": "api",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
should we be using yarn all through ? if yes then we will not need this file but a yarn variant

Reply:
ok, i will remove it